package-scripts/datadog-agent/postinst

```diff
@@ -31,7 +33,11 @@ if [ -f /etc/debian_version ] || [ "$LINUX_DISTRIBUTION" == "Debian" ] || [ "$LI
   set -e
   case "$1" in
       configure)
-          update-rc.d datadog-agent defaults
+          if which systemd >/dev/null 2>&1; then
+	    ln -s /lib/systemd/system/datadog-agent.service /etc/systemd/system/datadog-agent.service
+	  else
+	    update-rc.d datadog-agent defaults
+	  fi
```

Comment:
nitpick: the spacing looks a bit wonky here. Can we try to use 4 spaces everywhere?